From 2c65fc7c2295cd07342bea08761b09c169594d4f Mon Sep 17 00:00:00 2001
From: Laurent Bigonville <bigon@bigon.be>
Date: Mon, 28 Jul 2014 18:46:42 +0000
Subject: Revert "systemd: add WantedBy snippets"

This reverts commit 7adb50c267b1be18e11bd4c658274dbe590de485.

The .service files are already statically enabled, adding the Install
section and the WantedBy option is useless

Conflicts:
	systemd-units/plymouth-quit-wait.service.in

https://bugs.freedesktop.org/show_bug.cgi?id=80048
---
diff --git a/systemd-units/plymouth-halt.service.in b/systemd-units/plymouth-halt.service.in
index fd27e10..59a1912 100644
--- a/systemd-units/plymouth-halt.service.in
+++ b/systemd-units/plymouth-halt.service.in
@@ -9,6 +9,3 @@ ConditionKernelCommandLine=!plymouth.enable=0
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=shutdown --attach-to-session
 ExecStartPost=-@PLYMOUTH_CLIENT_DIR@/plymouth show-splash
 Type=forking
-
-[Install]
-WantedBy=halt.target
diff --git a/systemd-units/plymouth-kexec.service.in b/systemd-units/plymouth-kexec.service.in
index 8aa661d..5a39037 100644
--- a/systemd-units/plymouth-kexec.service.in
+++ b/systemd-units/plymouth-kexec.service.in
@@ -9,6 +9,3 @@ ConditionKernelCommandLine=!plymouth.enable=0
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=shutdown --attach-to-session
 ExecStartPost=-@PLYMOUTH_CLIENT_DIR@/plymouth show-splash
 Type=forking
-
-[Install]
-WantedBy=kexec.target
diff --git a/systemd-units/plymouth-poweroff.service.in b/systemd-units/plymouth-poweroff.service.in
index 305ece6..8ebaf76 100644
--- a/systemd-units/plymouth-poweroff.service.in
+++ b/systemd-units/plymouth-poweroff.service.in
@@ -9,5 +9,3 @@ ConditionKernelCommandLine=!plymouth.enable=0
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=shutdown --attach-to-session
 ExecStartPost=-@PLYMOUTH_CLIENT_DIR@/plymouth show-splash
 Type=forking
-[Install]
-WantedBy=poweroff.target
diff --git a/systemd-units/plymouth-quit-wait.service.in b/systemd-units/plymouth-quit-wait.service.in
index 89d9004..fbb235c 100644
--- a/systemd-units/plymouth-quit-wait.service.in
+++ b/systemd-units/plymouth-quit-wait.service.in
@@ -6,5 +6,3 @@ After=rc-local.service plymouth-start.service systemd-user-sessions.service
 ExecStart=-@PLYMOUTH_CLIENT_DIR@/plymouth --wait
 Type=oneshot
 TimeoutSec=0
-[Install]
-WantedBy=multi-user.target
diff --git a/systemd-units/plymouth-quit.service.in b/systemd-units/plymouth-quit.service.in
index 24c11bb..cf9901e 100644
--- a/systemd-units/plymouth-quit.service.in
+++ b/systemd-units/plymouth-quit.service.in
@@ -6,5 +6,3 @@ After=rc-local.service plymouth-start.service systemd-user-sessions.service
 ExecStart=-@PLYMOUTH_CLIENT_DIR@/plymouth quit
 Type=oneshot
 TimeoutSec=20
-[Install]
-WantedBy=multi-user.target
diff --git a/systemd-units/plymouth-read-write.service.in b/systemd-units/plymouth-read-write.service.in
index 5abfc49..55b975e 100644
--- a/systemd-units/plymouth-read-write.service.in
+++ b/systemd-units/plymouth-read-write.service.in
@@ -8,5 +8,3 @@ ConditionPathExists=!/etc/initrd-release
 [Service]
 ExecStart=-@PLYMOUTH_CLIENT_DIR@/plymouth update-root-fs --read-write
 Type=oneshot
-[Install]
-WantedBy=sysinit.target
diff --git a/systemd-units/plymouth-reboot.service.in b/systemd-units/plymouth-reboot.service.in
index ce56855..6d5309f 100644
--- a/systemd-units/plymouth-reboot.service.in
+++ b/systemd-units/plymouth-reboot.service.in
@@ -9,5 +9,3 @@ ConditionKernelCommandLine=!plymouth.enable=0
 ExecStart=@PLYMOUTH_DAEMON_DIR@/plymouthd --mode=shutdown --attach-to-session
 ExecStartPost=-@PLYMOUTH_CLIENT_DIR@/plymouth show-splash
 Type=forking
-[Install]
-WantedBy=reboot.target
diff --git a/systemd-units/plymouth-start.service.in b/systemd-units/plymouth-start.service.in
index 0d5ff2a..ddfb08f 100644
--- a/systemd-units/plymouth-start.service.in
+++ b/systemd-units/plymouth-start.service.in
@@ -12,5 +12,3 @@ ExecStartPost=-@PLYMOUTH_CLIENT_DIR@/plymouth show-splash
 Type=forking
 KillMode=none
 SendSIGKILL=no
-[Install]
-WantedBy=sysinit.target
--
cgit v0.9.0.2-2-gbebe
